middleware/logger/default_logger.go

```
@@ -31,7 +31,7 @@
 			}
 			buf.WriteString(
 				fmt.Sprintf("%s |%s %3d %s| %13v | %15s |%s %-7s %s| %-"+data.ErrPaddingStr+"s %s\n",
-					data.Timestamp.Load().(string), //nolint:forcetypeassert // Timestamp is always a string
+					data.Timestamp.Load().(string), //nolint:forcetypeassert,errcheck // Timestamp is always a string
```

golangci-lint

directive `//nolint:forcetypeassert,errcheck // Timestamp is always a string` is unused for linter "errcheck"

(nolintlint)